member_card/__init__.py

```diff
@@ -1,2 +1,52 @@
 from . import models
 from . import wizard
+
+
+from openupgradelib import openupgrade
+from odoo import api, SUPERUSER_ID
+
+
+renamed_view_xml_ids = (
+    (
```

Comment:
All of this should rather be put in a file `pre_init_hook.py` and then you import that file in this `__init__.py` file. Check out the examples in the code.